lib/internal/Magento/Framework/Validator/Email.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Email address validator
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\Validator;
+
+class Email extends AbstractValidator implements \Magento\Framework\Validator\ValidatorInterface
```

Comment:
I believe the plan was to avoid inheritance from `AbstractValidator` whenever possible, by just implementing a couple of `\Zend_Validate_Interface` methods.